### PR TITLE
Tweak: Make sure the Container Gap does not break BC [ED-11791]

### PR DESCRIPTION
**Pull request by louiswol94**

### CODE REVIEW

**Review by TzviRabinovitch — DISMISSED** [review]

File: assets/dev/js/editor/controls/gaps.js [hunk 1]
```diff
@@ -21,6 +21,30 @@ ControlGapItemView = ControlDimensionsView.extend( {
...
+			values[ key ] = value;
+		}
+
+		const conversion = this.model.get( 'conversion_map' );
+
+		if ( conversion && conversion.old_key && conversion.new_key ) {
+			values[ conversion.old_key ] = parseInt( values[ conversion.new_key ] );
+		}
+
+		this.setSettingsModel( values );
```
Comment (louiswol94) [inline assets/dev/js/editor/controls/gaps.js:42]:
```suggestion
		this.setSettingsModel( values );//
```

File: assets/dev/js/editor/controls/gaps.js [hunk 2]
```diff
@@ -21,6 +21,30 @@ ControlGapItemView = ControlDimensionsView.extend( {
...
+			values[ key ] = value;
+		}
+
+		const conversion = this.model.get( 'conversion_map' );
+
+		if ( conversion && conversion.old_key && conversion.new_key ) {
+			values[ conversion.old_key ] = parseInt( values[ conversion.new_key ] );
+		}
+
+		this.setSettingsModel( values );//
```
Comment (louiswol94) [inline assets/dev/js/editor/controls/gaps.js:42]:
```suggestion
		this.setSettingsModel( values );
```

File: assets/dev/js/editor/controls/gaps.js [hunk 3]
```diff
@@ -21,6 +21,30 @@ ControlGapItemView = ControlDimensionsView.extend( {
...
+
+	setValue( key, value ) {
+		var values = this.getControlValue();
+
+		if ( 'object' === typeof key ) {
+			_.each( key, function( internalValue, internalKey ) {
+				values[ internalKey ] = internalValue;
+			} );
+		} else {
+			values[ key ] = value;
```
Comment (louiswol94) [inline assets/dev/js/editor/controls/gaps.js:33]:
```suggestion
			values[ key ] = value;//
```

File: assets/dev/js/editor/controls/gaps.js [hunk 4]
```diff
@@ -21,6 +21,30 @@ ControlGapItemView = ControlDimensionsView.extend( {
...
+
+	setValue( key, value ) {
+		var values = this.getControlValue();
+
+		if ( 'object' === typeof key ) {
+			_.each( key, function( internalValue, internalKey ) {
+				values[ internalKey ] = internalValue;
+			} );
+		} else {
+			values[ key ] = value;//
```
Comment (louiswol94) [inline assets/dev/js/editor/controls/gaps.js:33]:
```suggestion
			values[ key ] = value;
```

File: assets/dev/js/editor/controls/gaps.js (hunk 3 above)
Comment (louiswol94) [inline assets/dev/js/editor/controls/gaps.js:33]:
```suggestion
			values[ key ] = value;//
```

File: assets/dev/js/editor/controls/gaps.js [hunk 5]
```diff
@@ -21,6 +21,30 @@ ControlGapItemView = ControlDimensionsView.extend( {
 			'column',
 		];
 	},
+
```
Comment (louiswol94) [inline assets/dev/js/editor/controls/gaps.js:24]: // dummy

Comment (louiswol94) [inline assets/dev/js/editor/controls/gaps.js:24]:
```suggestion
//dummy
```

File: assets/dev/js/editor/controls/gaps.js [hunk 6]
```diff
@@ -21,6 +21,30 @@ ControlGapItemView = ControlDimensionsView.extend( {
 			'column',
 		];
 	},
+//dummy
```
Comment (louiswol94) [inline assets/dev/js/editor/controls/gaps.js:24]:
```suggestion
```

File: assets/dev/js/editor/controls/gaps.js [hunk 7]
```diff
@@ -21,6 +21,29 @@ ControlGapItemView = ControlDimensionsView.extend( {
 			'column',
 		];
 	},
+	setValue( key, value ) {
```
Comment (louiswol94) [inline assets/dev/js/editor/controls/gaps.js:24]:
```suggestion
	setValue( key, value ) { // hi
```

File: assets/dev/js/editor/controls/gaps.js [hunk 8]
```diff
@@ -21,6 +21,29 @@ ControlGapItemView = ControlDimensionsView.extend( {
 			'column',
 		];
 	},
+	setValue( key, value ) { // hi
```
Comment (louiswol94) [inline assets/dev/js/editor/controls/gaps.js:24]:
```suggestion
	setValue( key, value ) {
```

File: assets/dev/js/editor/controls/gaps.js (hunk 7 above)
Comment (louiswol94) [inline assets/dev/js/editor/controls/gaps.js:24]:
```suggestion

	setValue( key, value ) {
```

**Review by barcovanrhijn — APPROVED** [review]

**Review by hein-obox — APPROVED** [review]

**Pull request merged**